build/util/is_on_path.py

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env python
+# Copyright 2023 The Cobalt Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script for checking if the current repo on the path."""
+
+import os
+import sys
+
+SRC_DIR = os.path.abspath(os.path.join(
+    os.path.dirname(__file__), os.pardir, os.pardir))
+
+def main():
+  abs_path_to_me = os.path.abspath(__file__)
+  rel_path_to_me = os.path.relpath(__file__, start=SRC_DIR)
```

Comment:
Can't we just check if SRC_DIR is in sys.path?

Reply:
No, we need to check specifically for the file to see if another cobalt repo is on the path in front of this one. I could theoretically look for any file in the cobalt repo but the check must be done nonetheless.

I could see `SRC_DIR` is _the_ first path on `PYTHONPATH` but that feels a little limiting.

Reply:
Got it